evennia/server/portal/portalsessionhandler.py

```diff
@@ -18,9 +18,11 @@
 # throttles
 _MAX_CONNECTION_RATE = float(settings.MAX_CONNECTION_RATE)
 _MAX_COMMAND_RATE = float(settings.MAX_COMMAND_RATE)
+_MAX_CHAR_LIMIT = settings.MAX_CHAR_LIMIT
```

Comment:
Better convert this to an int() here, so eventual unit errors in the settings file get caught early (doing > operations can have some unexpected results when mixing types).